pkg/operator/observe_config.go

```diff
@@ -97,6 +101,58 @@ func (c ConfigObserver) sync() error {
 	return nil
 }
 
+// observeClusterConfig observes cloud provider configuration from
+// cluster-config-v1 in order to configure kube-controller-manager's cloud
+// provider.
+func observeClusterConfig(kubeClient kubernetes.Interface, clientConfig *rest.Config, observedConfig map[string]interface{}) (map[string]interface{}, error) {
+	clusterConfig, err := kubeClient.CoreV1().ConfigMaps("kube-system").Get("cluster-config-v1", metav1.GetOptions{})
```

Comment:
I don't see the namespace scoped informer being plumbed to the control loop to watch for events.

Reply:
Fixed (I think)